adoc/quickstart3_chap_suma_keys_and_first_client.adoc

```diff
@@ -46,54 +46,67 @@ endif::[]
+For {productname}{nbsp}3 and later, you can choose to use either traditional or Salt clients, or a mixture of both, depending on your environment and requirements.
+
 
-Salt::
-Is an end-to-end data-center automation tool which may also be used outside the scope of {productname} to introduce reactive, real-time orchestration, and configuration management.
-Managed systems can coexist using both *traditional* and *Salt* frameworks.
-This functionality provides a safe learning environment when switching to *Salt* while you continue to maintain existing deployments.
 
 [[create.act.keys]]
 == Creating Activation Keys
 
-The following procedure will guide you through creating the activation keys for your test channels.
-Activation keys can be used with both Salt and traditional clients.
 
-Activation keys define the number of software entitlements, channels, and groups a client system is allowed to subscribe to.
-This information is passed on to all systems registered with said key.
-Each activation key is bound to the organization for which it has been created.
+Activation keys are used with both traditional and Salt clients to ensure that your clients have the correct software entitlements, are connecting to the appropriate channels, and are subscribed to the relevant groups.
+Each activation key is bound to an orgnization, which you can set when you create the key.
+
+This section contains information on how to create activation keys for both traditional and Salt clients, and provides some best practices for working with activation keys.
+
 
 [[create.activation.keys]]
 .Procedure: Creating Activation Keys
 . As the administrator login to the {productname} {webui}.
-. Switch to menu:Systems[Activation Keys].
-. Click the btn:[Create Key] link at the upper right corner.
+. Navigate to menu:Systems[Activation Keys].
+. Click the btn:[Create Key] button in the upper right corner.
 +
 
 image::systems_create_activation_key.png[scaledwidth=80%]
 
-. Enter an menu:Activation Key Details[Description] to identify the generated activation key.
-. Enter a human readable menu:Activation Key Details[Key] string value such as *SLES12-SP3* to represent the distribution and service pack associated with the key.
+. In the [guimenu]``Description`` field, enter a name for the activation key.
+. In the [guimenu]``Key`` field, enter the distribution and service pack associated with the key. For example, ``SLES12-SP3`` for {sls}{nbsp}12{nbsp}SP3.
 +
 
 .Allowed Characters
-WARNING: For all {suse}
-Products do not use commas within the key string.
-Red Hat Products require commas.
+WARNING: Do not use commas in the [guimenu]``Key`` field for any {suse} products.
+However, you *must* use commas for Red Hat Products.
 For more information, see <<ref.webui.systems.activ-keys>>.
 +
 
-. Select the {sle} channel that was added during
+. In the [guimenu]``Base Channels`` drop-down field, select the {sle} channel that you added during
```

Comment:
I think the style guide wants us to use drop-down box...

Reply:
OK :)